docs/checks.md

```diff
@@ -83,7 +83,7 @@ Different types of branch protection protect against different risks:
 
     - requiring two or more reviewers protects even more from the insider risk 
     whereby a compromised contributor can be used by an attacker to LGTM 
-    the attacker PR and inject a malicious code as if it was legitm.
```

Comment:
Please see our [CONTRIBUTING.md section](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#updating-docs) on modifying this file.

Reply:
Hey @spencerschrock, I added the change to the corresponding `checks.yaml` file